### PR TITLE
TR-3104 add workgroup option for athena queries - defaults to primary

### PR DESCRIPTION
**Pull request by dremsberg-msys**

### CODE REVIEW

**Comment by mtgray-sp** [comment]
> Is this fine? Or should this be "3.1.0" ? No material change - the new option has a default. Any services that want to specify the workgroup option will need to require this new version or higher.

I don't have a strong opinion about this, but if we're following semantic versioning, we should increment the minor version rather than the patch version. Incrementing the minor version means that we're adding backwards compatible new features. Incrementing the patch version means that we're fixing bugs in a backwards compatible way.

**Review by mtgray-sp — APPROVED** [review]

**Comment by dremsberg-msys** [comment]
> > Is this fine? Or should this be "3.1.0" ? No material change - the new option has a default. Any services that want to specify the workgroup option will need to require this new version or higher.
> 
> I don't have a strong opinion about this, but if we're following semantic versioning, we should increment the minor version rather than the patch version. Incrementing the minor version means that we're adding backwards compatible new features. Incrementing the patch version means that we're fixing bugs in a backwards compatible way.

I'll change this to 3.1.0   It's not a patch

**Review by SailakshmiPisupati — APPROVED** [review]

File: lib/athena/index.js [hunk 1]
```diff
@@ -27,7 +28,8 @@ class AthenaQueryClient {
         QueryString: sql,
         ResultConfiguration: {
           OutputLocation: `s3://${this.s3Bucket}`
-        }
+        },
+        WorkGroup: `${this.workgroup}`
```
Comment (ischer) [inline lib/athena/index.js:32]: can just make this `WorkGroup: this.workgroup`

Comment (SailakshmiPisupati) [reply lib/athena/index.js:32]: good catch!

**Pull request merged**